### PR TITLE
[CARE-6402] Support tiered pricing plans

### PR DESCRIPTION
**Pull request by marko-ogg**

### CODE REVIEW

File: src/types/Plan.ts [hunk 1]
```diff
@@ -47,8 +48,17 @@ export interface AddOn {
...
     currency: Currency;
-    amount: number;
+    billing_schema: BillingSchema;
+    amount: number | null;
     unit: string | null;
+    tiers_mode: TiersMode | null;
+    tiers: PricingTire[] | null;
+}
+
+export interface PricingTire {
```
Comment (kudlajz) [inline src/types/Plan.ts:58]:
```suggestion
export interface PricingTier {
```

Comment (marko-ogg) [reply src/types/Plan.ts]: https://github.com/prezly/javascript-sdk/pull/337/commits/46132b12f606baad78fdad56a3f1728a79e2c023

**Review by kudlajz — APPROVED** [review]

File: src/types/Plan.ts [hunk 2]
```diff
@@ -47,8 +48,11 @@ export interface AddOn {
 export interface PricingPlan {
     billing_cycle: BillingCycle;
     currency: Currency;
-    amount: number;
+    billing_schema: BillingSchema;
```
Comment (kudlajz) [inline src/types/Plan.ts:51]: Just noticed this when testing, but API returns this as `billing_scheme` and not `billing_schema`. Not sure which form is the correct one so I'll let you fix it.

Comment (marko-ogg) [reply src/types/Plan.ts:51]: I see: https://github.com/prezly/javascript-sdk/pull/339

**Pull request merged**